src/models/game/reducer.js

```diff
@@ -26,60 +27,75 @@ const initialState = {
 const gameReducer = (state = initialState, action) => {
   switch (action.type) {
     case gameStarted.type: {
+      return {
+        ...state,
+        phase: (state.phase = 1),
+      };
+    }
+    case deckCreated.type: {
       return {
         ...state,
         ...action.payload,
       };
     }
-
-    case playerFolded.type: {
+    case cardsDealt.type: {
       return {
         ...state,
         ...action.payload,
       };
     }
 
-    case betRaised.type: {
+    case betsPlaced.type: {
       return {
         ...state,
         ...action.payload,
       };
     }
 
-    case cardReplaced.type: {
+    case playerFolded.type: {
+      return {
+        ...state,
+        phase: (state.phase = 4),
```

Comment:
Typo?

Reply:
Thought I had to reference state.phase, Fixed.